app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss

```diff
@@ -152,6 +152,10 @@
 
       &.is-image {
         @apply rounded-lg;
+
+        .message__mail-head {
+          @apply px-4 py-2;
+        }
```

Comment:
Let's not use `@apply` syntax for new changes

Reply:
@scmmishra, Is there a specific reason behind this? In this SCSS file, the conversation view style is written. The issue is that we need to add padding for the text if there is an attachment message bubble. However, for the attachment message bubble, there is no padding. If we want to move it I can pick it as a separate task. I think it needs some effort.

Reply:
Noted, that sounds great to me. No particular reason except using `@apply` in tailwind is generally considered an anti-pattern, lot of corner cases which are not documented. The author himself suggests using `theme` instead. [ref](https://twitter.com/adamwathan/status/1559250403547652097?lang=en) 

Also, from a tech-stack PoV, one of the benefits tailwind has is not having to manage multiple CSS files. 

> If we want to move it I can pick it as a separate task. I think it needs some effort.

Sure, makes sense.

Reply:
Resolving the conversation for now

Reply:
@scmmishra Yeah, Our clean-up from SCSS to tailwind is still spending. We have prioritized the dark mode.